includes/admin/class-sensei-learner-management.php

```diff
@@ -752,38 +752,38 @@ public function add_learner_notices() {
 				case 'error_enrol':
 					$notice = [
 						'error',
-						__( 'An error occurred while manually enrolling the learner.', 'sensei-lms' ),
+						__( 'An error occurred while enrolling the learner.', 'sensei-lms' ),
 					];
 					break;
 				case 'error_enrol_multiple':
 					$notice = [
 						'error',
-						__( 'An error occurred while manually enrolling the learners.', 'sensei-lms' ),
+						__( 'An error occurred while enrolling the learners.', 'sensei-lms' ),
 					];
 					break;
 				case 'error_withdraw':
 					$notice = [
 						'error',
-						__( 'An error occurred removing the learner\'s manual enrollment', 'sensei-lms' ),
+						__( 'An error occurred removing the learner\'s enrollment', 'sensei-lms' ),
```

Comment:
While we're here, might as well add a period at the end of the sentence like the others. 🙂

Reply:
Good catch 🦅! Fixed here: https://github.com/Automattic/sensei/pull/3466/commits/08808a49d175d7a7a2d793de178499ca534327b9